full-service/src/service/models/transaction_memo.rs

```diff
@@ -163,3 +190,102 @@ impl TryFrom<&Account> for TransactionMemoSignerCredentials {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use mc_account_keys::AccountKey;
+    use rand::{rngs::StdRng, SeedableRng};
+
+    #[test]
+    fn test_rth_memo_b58_roundtrip() {
```

Comment:
⛏️ this may just be a me thing but the `test_` prefix seems unnecessarily redundant, the module is `tests`, there is a macro before called `#[test]`.
though it seems to be used often so 🤷 

```suggestion
    fn rth_memo_b58_roundtrip() {
```